src/main/resources/beam-template.conf

```diff
@@ -273,6 +273,7 @@ beam.agentsim.agents.freight {
   plansFilePath = ${beam.inputDirectory}"/freight/payload-plans.csv"
   toursFilePath = ${beam.inputDirectory}"/freight/freight-tours.csv"
   carriersFilePath = ${beam.inputDirectory}"/freight/freight-carriers.csv"
+  vehicleTypesFilePath = ""
```

Comment:
Maybe we could use
`#@optional
carrierParkingFilePath = string`
in order to make it more clear that this is optional?

Reply:
good idea!

Reply:
If I understand correctly we need to run `generateConfig` task again. It generates an Option for this field. We also need to slightly modify the code (it may get even simpler).